<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>VentanaAjustes</class>
 <widget class="QWidget" name="VentanaAjustes">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>298</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>400</width>
    <height>298</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>400</width>
    <height>298</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Pollock Effect - Costa Rica Institute of Technology</string>
  </property>
  <property name="windowIcon">
   <iconset resource="images.qrc">
    <normaloff>:/images/icon.png</normaloff>:/images/icon.png</iconset>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>400</width>
     <height>300</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="images.qrc">:/images/ventanaAjustes.jpg</pixmap>
   </property>
  </widget>
  <widget class="QFrame" name="frame_click_hsv">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>250</y>
     <width>161</width>
     <height>33</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QLabel" name="label_33">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>11</width>
      <height>13</height>
     </rect>
    </property>
    <property name="text">
     <string>H:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_34">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>10</y>
      <width>10</width>
      <height>13</height>
     </rect>
    </property>
    <property name="text">
     <string>S:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_35">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>10</y>
      <width>10</width>
      <height>13</height>
     </rect>
    </property>
    <property name="text">
     <string>V:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_click_h">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>10</y>
      <width>31</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_click_s">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>10</y>
      <width>31</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_click_v">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>10</y>
      <width>31</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
   <zorder>label_33</zorder>
   <zorder>label_click_h</zorder>
   <zorder>label_click_s</zorder>
   <zorder>label_click_v</zorder>
   <zorder>label_35</zorder>
   <zorder>label_34</zorder>
  </widget>
  <widget class="QSlider" name="barra_erode_der">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>70</y>
     <width>160</width>
     <height>19</height>
    </rect>
   </property>
   <property name="maximum">
    <number>25</number>
   </property>
   <property name="pageStep">
    <number>5</number>
   </property>
   <property name="value">
    <number>1</number>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QSpinBox" name="h_min_der">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>140</y>
     <width>42</width>
     <height>22</height>
    </rect>
   </property>
   <property name="maximum">
    <number>500</number>
   </property>
   <property name="value">
    <number>99</number>
   </property>
  </widget>
  <widget class="QLabel" name="label_28">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>170</y>
     <width>6</width>
     <height>13</height>
    </rect>
   </property>
   <property name="text">
    <string>S</string>
   </property>
  </widget>
  <widget class="QSlider" name="barra_erode_izq">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>70</y>
     <width>160</width>
     <height>19</height>
    </rect>
   </property>
   <property name="maximum">
    <number>25</number>
   </property>
   <property name="pageStep">
    <number>5</number>
   </property>
   <property name="value">
    <number>1</number>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QLabel" name="label_20">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>50</y>
     <width>46</width>
     <height>13</height>
    </rect>
   </property>
   <property name="text">
    <string>1</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_24">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>170</y>
     <width>6</width>
     <height>13</height>
    </rect>
   </property>
   <property name="text">
    <string>S</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_19">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>50</y>
     <width>46</width>
     <height>13</height>
    </rect>
   </property>
   <property name="text">
    <string>Erosion</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="v_max_izq">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>200</y>
     <width>42</width>
     <height>22</height>
    </rect>
   </property>
   <property name="maximum">
    <number>500</number>
   </property>
   <property name="value">
    <number>255</number>
   </property>
  </widget>
  <widget class="QSpinBox" name="s_min_izq">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>170</y>
     <width>42</width>
     <height>22</height>
    </rect>
   </property>
   <property name="maximum">
    <number>500</number>
   </property>
   <property name="value">
    <number>150</number>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_hsv_ajustes">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>250</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Open HSV</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="v_min_izq">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>200</y>
     <width>42</width>
     <height>22</height>
    </rect>
   </property>
   <property name="maximum">
    <number>500</number>
   </property>
   <property name="value">
    <number>130</number>
   </property>
  </widget>
  <widget class="QSpinBox" name="s_min_der">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>170</y>
     <width>42</width>
     <height>22</height>
    </rect>
   </property>
   <property name="maximum">
    <number>500</number>
   </property>
   <property name="value">
    <number>195</number>
   </property>
  </widget>
  <widget class="QSpinBox" name="s_max_izq">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>170</y>
     <width>42</width>
     <height>22</height>
    </rect>
   </property>
   <property name="maximum">
    <number>500</number>
   </property>
   <property name="value">
    <number>230</number>
   </property>
  </widget>
  <widget class="QLabel" name="label_26">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>140</y>
     <width>7</width>
     <height>13</height>
    </rect>
   </property>
   <property name="text">
    <string>H</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_cerrar_hsv">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>250</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Close HSV</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_23">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>140</y>
     <width>7</width>
     <height>13</height>
    </rect>
   </property>
   <property name="text">
    <string>H</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_18">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>50</y>
     <width>46</width>
     <height>13</height>
    </rect>
   </property>
   <property name="text">
    <string>1</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="s_max_der">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>170</y>
     <width>42</width>
     <height>22</height>
    </rect>
   </property>
   <property name="maximum">
    <number>500</number>
   </property>
   <property name="value">
    <number>215</number>
   </property>
  </widget>
  <widget class="QLabel" name="label_30">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>230</y>
     <width>46</width>
     <height>13</height>
    </rect>
   </property>
   <property name="text">
    <string>Max</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_29">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>230</y>
     <width>46</width>
     <height>13</height>
    </rect>
   </property>
   <property name="text">
    <string>Min</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_39">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>110</y>
     <width>94</width>
     <height>13</height>
    </rect>
   </property>
   <property name="text">
    <string>Color para detectar</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="v_max_der">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>200</y>
     <width>42</width>
     <height>22</height>
    </rect>
   </property>
   <property name="maximum">
    <number>500</number>
   </property>
   <property name="value">
    <number>255</number>
   </property>
  </widget>
  <widget class="QLabel" name="label_41">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>230</y>
     <width>46</width>
     <height>13</height>
    </rect>
   </property>
   <property name="text">
    <string>Max</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_21">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>110</y>
     <width>94</width>
     <height>13</height>
    </rect>
   </property>
   <property name="text">
    <string>Color para detectar</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_17">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>50</y>
     <width>46</width>
     <height>13</height>
    </rect>
   </property>
   <property name="text">
    <string>Erosion</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="h_min_izq">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>140</y>
     <width>42</width>
     <height>22</height>
    </rect>
   </property>
   <property name="maximum">
    <number>500</number>
   </property>
   <property name="value">
    <number>25</number>
   </property>
  </widget>
  <widget class="QSpinBox" name="h_max_izq">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>140</y>
     <width>42</width>
     <height>22</height>
    </rect>
   </property>
   <property name="maximum">
    <number>500</number>
   </property>
   <property name="value">
    <number>50</number>
   </property>
  </widget>
  <widget class="QSpinBox" name="v_min_der">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>200</y>
     <width>42</width>
     <height>22</height>
    </rect>
   </property>
   <property name="maximum">
    <number>500</number>
   </property>
   <property name="value">
    <number>150</number>
   </property>
  </widget>
  <widget class="QLabel" name="label_40">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>230</y>
     <width>46</width>
     <height>13</height>
    </rect>
   </property>
   <property name="text">
    <string>Min</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="h_max_der">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>140</y>
     <width>42</width>
     <height>22</height>
    </rect>
   </property>
   <property name="maximum">
    <number>500</number>
   </property>
   <property name="value">
    <number>110</number>
   </property>
  </widget>
  <widget class="QLabel" name="label_27">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>200</y>
     <width>6</width>
     <height>13</height>
    </rect>
   </property>
   <property name="text">
    <string>V</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_25">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>200</y>
     <width>6</width>
     <height>13</height>
    </rect>
   </property>
   <property name="text">
    <string>V</string>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="images.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>barra_erode_izq</sender>
   <signal>valueChanged(int)</signal>
   <receiver>label_18</receiver>
   <slot>setNum(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>308</x>
     <y>130</y>
    </hint>
    <hint type="destinationlabel">
     <x>318</x>
     <y>111</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>barra_erode_der</sender>
   <signal>valueChanged(int)</signal>
   <receiver>label_20</receiver>
   <slot>setNum(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>517</x>
     <y>129</y>
    </hint>
    <hint type="destinationlabel">
     <x>528</x>
     <y>106</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
